snooty/rstparser.py

```diff
@@ -687,6 +687,10 @@ def run(self) -> List[docutils.nodes.Node]:
 
             return [node]
 
+        # these directives should not be treated like a tabs directive
+        if self.name == "tabs-pillstrip" or self.name == "tabs-selector":
```

Comment:
You can just do `if self.name in {"tabs-pillstrip", "tabs-selector"}` to make the intent slightly more clear